tests/MyGame/Example/Monster.cs

```diff
@@ -129,6 +129,33 @@ public sealed class Monster : Table {
     return new Offset<Monster>(o);
   }
   public static void FinishMonsterBuffer(FlatBufferBuilder builder, Offset<Monster> offset) { builder.Finish(offset.Value, "MONS"); }
+  
+  public static int KeysCompare(string o1, string o2) { return o1.CompareTo(o2); }
+
+  public int KeyCompareWithValue(string val) { return Name.CompareTo(val); }
```

Comment:
Maybe an idea for a future improvement: Since `Name` allocates a string, if we could somehow compare directly against what is in the byte array, this would be faster. Though I guess that means `val` would have to be supplied as UTF-8 bytes as well.